vcluster/configure/vcluster-yaml/sync/from-host/secrets.mdx

```diff
@@ -0,0 +1,159 @@
+---
+title: Secrets
+sidebar_label: secrets
+sidebar_position: 11
+description: Configuration for ...
+---
+
+import EnableSwitch from '../../../../_partials/config/sync/fromHost/secrets.mdx'
+import ProAdmonition from '../../../../_partials/admonitions/pro-admonition.mdx'
+import FromHostSecretExample from '../../../../_fragments/sync-from-host-secret-example.mdx'
+
+By default, this is turned off.
+
+:::info No need to configure RBAC
+vCluster automatically adds the required cluster RBAC permissions for retrieving the Secrets and syncing the resources from the host to the virtual cluster.
+:::
+
+Enabling this allows you to sync Secrets from the specified namespaces in the host to the specified namespaces in the vCluster.
+```yaml title="vcluster.yaml"
+sync:
+  fromHost:
+    secrets:
+      enabled: true
+      selector:
+        mappings:
+          # syncs all Secrets from "foo" namespace
+          # to the "bar" namespace in vCluster. Secrets names are unchanged.
+          "foo/*": "bar/*"
+```
+
```

Comment:
Would be good to add a short paragraph, something like:
Here are a few things to remember when configuring the sync.